package.json

```diff
@@ -29,6 +29,9 @@
     "cheerio": "^1.0.0-rc.3",
     "discord.js": "^12.1.1",
     "jest": "^26.0.1",
+    "jquery": "^3.5.1",
+    "jsdom": "^16.4.0",
+    "moment": "^2.28.0",
```

Comment:
Using moment to list watch expirations as how much time is left rather than a fixed date.  Moment is dying soon RIP.

Reply:
I've been using dayjs and it's very light weight and similar, so that's an option for the future.  Marking resolved for now.